### PR TITLE
Tests: Add for api module

### PR DESCRIPTION
**Pull request by SchoolGuy**
Currently, the API module doesn't have tests. This PR aims to introduce a few basic tests.

This is a splitout of #209

### CODE REVIEW

**Comment by SchoolGuy** [comment]
The tests will be red on purpose as they reflect the bugs that have been found through writing them.

File: orthos2/api/tests/test_forms.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    fixtures = ["orthos2/data/fixtures/serialconsoletypes.json"]
+
+    def test_form(self):
+        """Test the serial console creation API form"""
+        # Arrange & Act
+        form = SerialConsoleAPIForm({
+            "stype": "1",
+            "baud_rate": "57600",
+            "kernel_device": "ttyS",
+            "kernel_device_num": "5"
```
Comment (watologo1) [inline orthos2/api/tests/test_forms.py:97]: I would use a valid kernel_device_num for ttySX.
x86 typically have ttyS0 and ttyS1.

File: orthos2/data/fixtures/tests/test_machines.json [hunk 2]
```diff
@@ -56,6 +56,7 @@
       "ipmi": true,
       "vm_capable": true,
       "vm_dedicated_host": true,
+      "virt_api_int": 0,
```
Comment (watologo1) [inline orthos2/data/fixtures/tests/test_machines.json:59]: what is this for?

**Review by watologo1 — APPROVED** [review]

File: orthos2/api/tests/test_forms.py (hunk 1 above)
Comment (SchoolGuy) [reply orthos2/api/tests/test_forms.py:97]: While you are correct that generally 0 or 1 is used, 5 is fully valid tty as well.

File: orthos2/data/fixtures/tests/test_machines.json (hunk 2 above)
Comment (SchoolGuy) [reply orthos2/data/fixtures/tests/test_machines.json:59]: To reference the virt API type.

**Pull request merged**